repo-scripts/size-analysis/bundle-definitions/functions.json

```diff
@@ -22,11 +22,12 @@
             "path": "functions",
             "imports": [
               "getFunctions",
-              "httpsCallable"
+              "httpsCallable",
+              "httpsCallableFromURL"
```

Comment:
Doesn't need to be added here, this represents a typical bundle of imports for a use case and it's not a common case that a developer would use httpsCallable and httpsCallableFromURL in the same app. In any case, since httpsCallable wraps the core code of httpsCallableFromURL, the bundle size should be about the same.